test/quantization/test_quantize_script.py

```diff
@@ -2615,3 +2617,29 @@ def forward(self, x):
         for x, obs in model._modules._c.items():
             graph_params.append((obs.getattr('3_scale_0'), obs.getattr('3_zero_point_0')))
         self.assertEqual(ref_qparams, graph_params)
+
+class TestQuantizeQATScript(QuantizationTestCase):
+
+    def test_prepare_qat(self):
```

Comment:
maybe we don't need to explicitly test for this since it is identical with static

Reply:
I assume this is a place holder for other tests

Reply:
it might make more sense in a PR or two, adding it here mostly for easy devex.  I'm flexible.  It will call a different pass for conv-bn in the future, so not exactly the same as static.

Reply:
I think you can just test for the fold bn functionality, that is orthogonal to what is been tested here